src/core/connection.h

```diff
@@ -175,6 +175,16 @@ typedef union QUIC_CONNECTION_STATE {
         //
         BOOLEAN LocalInterfaceSet : 1;
 
+        //
+        // This flag indicates that the connection should ignore the fixed bit on received packets.
+        //
+        BOOLEAN IgnoreFixedBit : 1;
+
+        //
+        // This value of the fixed bit on send packets.
+        //
+        BOOLEAN FixedBit : 1;
```

Comment:
See the comment at the top of this union. We need to keep this copy in sync with the one in quictypes.h.